ci/scripts/lint.sh

```diff
@@ -45,8 +43,10 @@ for dir in $dirs; do
         ./gradlew build
       fi
     elif [[ "$dir" =~ "-typescript" ]]; then
-      print "Running TSLint"
-      tslint --project .
+        print "Installing node modules"
+        npm install
+        print "Running Lint"
+        npm run lint
```

Comment:
It would be good to combine this into a single test with "-javascript" since exactly the same operation should be carried out for both project types